config/locales/en.yml

```diff
@@ -36,6 +36,8 @@ en:
         create: "Add"
   activerecord:
     attributes:
+      gsa18f/procurement:
+        recurring: "Recurring charge"
```

Comment:
Well, my thinking is that we put all _overrides_ for fields and such in i18n files...I don't think we need to do have entries for every single piece of text that gets displayed in the app. Again, the goal with this PR was making the use case form templates easy to read/change.